### DISCUSSION TITLE
How to restart after Ctrl+ C?

### DISCUSSION THREAD

**Discussion by mcamou** [Sithuk/ubuntu-server-zfsbootmenu#22] [General]
Hello and thanks for building this project.

I've had to Ctrl+C because I noticed in the middle of the run that I'd misconfigured something. When restarting I got an error stating that rpool was mounted. I tried `zpool export rpool` and the system said that rpool is busy. `lsof` shows no files using the directory, but `zfs umount -a` also says that it's busy. The only option I found was rebooting. Any ideas?

**Comment by Sithuk** [comment]
I haven't had much success in exporting a mounted pool even with the force flag on zpool export. You did what I would have done which is to re-boot and start again.

**Reply by mcamou** [reply]
OK, I thought you might have figured out a way of doing it. Thank you.

**Reply by ExpatUK** [reply]
A bit late to the party here, but this is how I've been getting around it after failed installs etc, so I thought it might be useful to reply here for posterity:
``` 
umount -lf /mnt/ub_server
zpool destroy rpool -f
```
The installer should then run again without too much issue.

Also worthy of note, on one particular server model (Dell R710) we managed to softbrick the bios due to excessive efi boot entries, requiring a jumper to be bridged to clear it. If anyone else runs into this issue (or just wants to keep a handle on excessive efi boot entries), you can delete the entries before reboot (and avoiding the softbrick) by deleting them with efibootmgr:
```
root@ubuntu-server:/tmp# efibootmgr
BootCurrent: 0007
Timeout: 0 seconds
BootOrder: 000B,000A,0009,0008,0007,0000,0001,0002,0003,0004,0005,0006
Boot0000* PLDS DVD+/-RW DS-8A9SH
Boot0001* Windows Boot Manager
Boot0002* EFI Network 1
Boot0003* EFI Network 2
Boot0004* EFI Network 3
Boot0005* EFI Network 4
Boot0006* EFI Fixed Disk Boot Device 1
Boot0007* DataTraveler 3.0
Boot0008* rEFInd Boot Manager
Boot0009* rEFInd Boot Manager Backup 1
Boot000A* rEFInd Boot Manager Backup 2
Boot000B* rEFInd Boot Manager Backup 3
 
root@ubuntu-server:/tmp# efibootmgr -b B -B
BootCurrent: 0007
Timeout: 0 seconds
BootOrder: 000A,0009,0008,0007,0000,0001,0002,0003,0004,0005,0006
Boot0000* PLDS DVD+/-RW DS-8A9SH
Boot0001* Windows Boot Manager
Boot0002* EFI Network 1
Boot0003* EFI Network 2
Boot0004* EFI Network 3
Boot0005* EFI Network 4
Boot0006* EFI Fixed Disk Boot Device 1
Boot0007* DataTraveler 3.0
Boot0008* rEFInd Boot Manager
Boot0009* rEFInd Boot Manager Backup 1
Boot000A* rEFInd Boot Manager Backup 2
 
root@ubuntu-server:/tmp# efibootmgr -b A -B
BootCurrent: 0007
Timeout: 0 seconds
BootOrder: 000A,0009,0008,0007,0000,0001,0002,0003,0004,0005,0006
Boot0000* PLDS DVD+/-RW DS-8A9SH
Boot0001* Windows Boot Manager
Boot0002* EFI Network 1
Boot0003* EFI Network 2
Boot0004* EFI Network 3
Boot0005* EFI Network 4
Boot0006* EFI Fixed Disk Boot Device 1
Boot0007* DataTraveler 3.0
Boot0008* rEFInd Boot Manager
Boot0009* rEFInd Boot Manager Backup 1
 
# etc...
```
